spec/services/rating_books/create_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ::RatingBooks::Create do
+  subject { described_class.call(user:, book:, rating: other_rating) }
+
+  let(:user)          { create(:user) }
+  let(:book)          { create(:book).decorate }
+  let(:rating)        { create(:rating) }
+  let(:other_rating)  { create(:rating) }
+  let(:create_rating) { described_class.call(user:, book:, rating:) }
+
+  before(:each) do
+    create_rating
+  end
+
+  context 'when user' do
```

Comment:
this context description is not clear. 
use 'when book is rated by user' or something like that.